worksheet_D/oxo.py

```diff
@@ -0,0 +1,117 @@
+class OxoBoard:
+    def __init__(self, x, y):
+        self.x = x
+        self.y = y
+        self.board = {(0, 0): 0,(1, 0): 0, (2, 0): 0,
+                      (0, 1): 0, (1, 1): 0, (2, 1): 0,
+                      (0, 2): 0, (1, 2): 0, (2, 2): 0}      # Creates dictionary.
```

Comment:
Rather than writing this out, how about creating an empty dictionary and then populating it using a loop (or loops)?